tm/src/main/java/io/seata/tm/api/DefaultGlobalTransaction.java

```diff
@@ -108,8 +113,34 @@ public void begin(int timeout, String name) throws TransactionException {
 
     }
 
+    @Override
+    public void begin(int timeout, String name, Propagation propagation) throws TransactionException {
+        this.propagation = propagation;
+        switch (this.propagation) {
+            case NOT_SUPPORTED:
+                RootContext.unbind();
+                return;
+            case REQUIRES_NEW:
+                RootContext.unbind();
```

Comment:
globalTransaction1.begin(6000,"xxx1");
DML1
globalTransaction1.begin(6000,"xxx2", Propagation. REQUIRES_NEW);
DML2
globalTransaction2.commit;
globalTransaction1.commit;    where is the xid1?